x/rollapp/keeper/rollapp.go

```diff
@@ -39,6 +39,10 @@ func (k Keeper) CheckAndUpdateRollappFields(ctx sdk.Context, update *types.MsgUp
 		current.InitialSequencer = update.InitialSequencer
 	}
 
+	if 0 < update.MinSequencerBond {
```

Comment:
we should enforce a minimum I blv

Reply:
if we want we can always set it to 0 but should be param like before

Reply:
fixed
0 would be weird because of kick threshold I think